### PR TITLE
Finished Feature/var.x

### LINKED ISSUE #37: enable var.x application
add var.x to the ctests
- use `BUMP` covariance in the yaml
- use the sample observations that are already in this repository

### PR DESCRIPTION
**Pull request by loganchen39**
@travissluka Passed all tests. Please check and approve. 

Close #37 .

### CODE REVIEW

File: test/testinput/var.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        west: -180
+    test: on
+    online diagnostics:
+      write increment: true
+    increment:
+      datadir: Data
+      filename: Data/var.output.increment.nc  # Ligang: Must have, Data/ is needed.
+      date: 2018-04-15T12:00:00Z
+      exp: 3dvar.iter1
+      type: in
```
Comment (travissluka) [inline test/testinput/var.yml]: remove unused parameters

```suggestion
    increment:
      filename: Data/var.output.increment.nc
```

File: test/testinput/var.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    departures: oman
+
+
+output:  # Ligang: must have.
+  datadir: Data
+  filename: var.output.final.nc
+  exp: 3dvar
+# frequency: PT6H
+  frequency: PT24H
+  type: an
```
Comment (travissluka) [inline test/testinput/var.yml]:
```suggestion
output:
  filename: Data/var.output.final.nc
```

**Review by travissluka — APPROVED** [review]
looks good, you can tell Jim we have a "working" VAR now! :-)

I've noted a couple of parameters in the yaml that are not actually used and should be deleted for clarity (They are by other models, but not by us given the way we currently generate filenames).

**Comment by loganchen39** [comment]
@travissluka Updated as suggested.

**Pull request merged**